store/copr/coprocessor_test.go

```diff
@@ -289,3 +290,101 @@ func rangeEqual(t *testing.T, ranges []kv.KeyRange, keys ...string) {
 		require.Equal(t, string(r.EndKey), keys[2*i+1])
 	}
 }
+
+func TestBuildPagingTasks(t *testing.T) {
+	t.Parallel()
+	// nil --- 'g' --- 'n' --- 't' --- nil
+	// <-  0  -> <- 1 -> <- 2 -> <- 3 ->
+	mockClient, cluster, pdClient, err := testutils.NewMockTiKV("", nil)
+	require.NoError(t, err)
+	defer func() {
+		pdClient.Close()
+		err = mockClient.Close()
+		require.NoError(t, err)
+	}()
+
+	_, regionIDs, _ := testutils.BootstrapWithMultiRegions(cluster, []byte("g"), []byte("n"), []byte("t"))
+	pdCli := &tikv.CodecPDClient{Client: pdClient}
+	defer pdCli.Close()
+
+	cache := NewRegionCache(tikv.NewRegionCache(pdCli))
+	defer cache.Close()
+
+	bo := backoff.NewBackofferWithVars(context.Background(), 3000, nil)
+
+	req := &kv.Request{}
+	req.Paging = true
+	flashReq := &kv.Request{}
+	flashReq.StoreType = kv.TiFlash
+	tasks, err := buildCopTasks(bo, cache, buildCopRanges("a", "c"), req, nil)
+	require.NoError(t, err)
+	require.Len(t, tasks, 1)
+	require.Len(t, tasks, 1)
+	taskEqual(t, tasks[0], regionIDs[0], "a", "c")
+	require.True(t, tasks[0].paging)
+	require.Equal(t, tasks[0].pagingSize, minPagingSize)
+}
+
+func TestCalculateTODO(t *testing.T) {
```

Comment:
This name doesn't match the function name. I choose 'ToDo' among 'todo', 'TODO' and 'ToDO'.

Reply:
Actually, I prefer the name `calculateRemain`, however it's used as calculating retry ranges, what about this change:
`calculateRemain` -> `calculateRetry`
`calculateToDo` -> `calculateRemain`